models/repo.go

```diff
@@ -1246,6 +1246,13 @@ func createRepository(e *xorm.Session, u *User, repo *Repository) (err error) {
 			return fmt.Errorf("getOwnerTeam: %v", err)
 		} else if err = t.addRepository(e, repo); err != nil {
 			return fmt.Errorf("addRepository: %v", err)
+		} else if err = PrepareWebhooks(repo, HookEventRepository, &api.RepositoryPayload{
```

Comment:
This will result `database table lock` on sqlite since we have a transaction here and another in `PrepareWebhooks`. It seems you have to create a `prepareWebhooks(e Engine...` method to avoid that.

Reply:
Fixed